content/en/docs/spinnaker-user-guides/using-dinghy.md

```diff
@@ -79,6 +82,7 @@ Here is an example Dinghyfile:
 Make sure you specify the following fields so that the Dinghyfile can create a pipeline with stages:
 
 * `.application`: The name of the application where pipelines will be created or updated.  If the application does not exist, it will be created.
+* `appmetadata`: Key/value pairs where you can define application metadata, such as whether Canary analysis is enabled (`"canaryConfigs": "true"`). You can retrieve a list of potential configs by opening the developer console for your browser when viewing the **Config** page of an application. Look for the possible parameters under the `spinnaker.application.attributes` key. Note that no validation is done for this field. 
```

Comment:
This suggestion adds a few more descriptions for configuration options on `appmetadata`

```suggestion
* `.globals`: Configuration used to define Top-level variables ([see Top-level variables](#top-level-variables))
* `.globals.save_app_on_update`:  Flag that allow to update application properties (see [see Application updates](#application-updates) for reference)
* `.appmetadata`: Key/value pairs where you can define application metadata, such as whether Canary analysis is enabled. You can retrieve a list of potential configs by opening the developer console for your browser when viewing the **Config** page of an application. Look for the possible parameters under the `spinnaker.application.attributes` key. Note that no validation is done for this field.
* `.appmetadata.dataSources.enabled`:  An array of features that will be displayed on Spinnaker
* `.appmetadata.dataSources.disabled`: An array of features that will not be displayed on Spinnaker
* `.appmetadata.customBanners[*]`: An array of banners that will be displayed on Spinnaker, must only have one item checked as enabled
* `.appmetadata.cloudProviders[*]`: An array of cloud providers to be used on the application```